api/v1/controllers/RSVPController.js

```diff
@@ -97,7 +100,7 @@ router.post('/attendee', middleware.request(requests.RSVPRequest),
 router.get('/attendee/', middleware.permission(roles.ATTENDEE), fetchRSVPByUser);
 router.get('/attendee/:id(\\d+)', middleware.permission(roles.ORGANIZERS), fetchRSVPById);
 router.put('/attendee/', middleware.request(requests.RSVPRequest),
-  middleware.permission(roles.ATTENDEE), updateRSVPByUser);
+  middleware.permission(roles.ATTENDEE, _isAuthenticated), updateRSVPByUser);
```

Comment:
Make sure there aren't other routers in this controller that use the isAuthenticated method that get effected by the changes.